src/documentation/runners/direct.md

```diff
@@ -33,9 +47,11 @@ You must specify your dependency on the Direct Runner.
 </dependency>
 ```
 
+<span class="language-python">When using Python, you do not need to specify your dependency on the Direct Runner.</span>
+
 ## Pipeline options for the Direct Runner
 
-When executing your pipeline from the command-line, set `runner` to `direct`. The default values for the other pipeline options are generally sufficient.
+When executing your pipeline from the command-line, set `runner` to `direct` or `DirectRunner`. The default values for the other pipeline options are generally sufficient.
 
 See the reference documentation for the  <span class="language-java">[`DirectOptions`]({{ site.baseurl }}/documentation/sdks/javadoc/{{ site.release_latest }}/index.html?org/apache/beam/runners/direct/DirectOptions.html)</span><span class="language-python">[`PipelineOptions`](https://github.com/apache/beam/blob/master/sdks/python/apache_beam/utils/pipeline_options.py)</span> interface (and its subinterfaces) for defaults and the complete list of pipeline configuration options.
```

Comment:
There is also `DirectOptions`, that is probably the better link here. https://beam.apache.org/documentation/sdks/pydoc/0.6.0/apache_beam.utils.html#apache_beam.utils.pipeline_options.DirectOptions

(Assuming that we are talking about direct runner specific options.)